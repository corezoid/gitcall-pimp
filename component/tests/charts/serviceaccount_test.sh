#!/usr/bin/env bash
$CZ helm template | sed -n '/serviceaccount\.yaml/,/Source\:/p'
