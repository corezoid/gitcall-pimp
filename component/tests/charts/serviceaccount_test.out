# Source: gitcall_pimp_release/charts/gitcall_pimp/templates/serviceaccount.yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: RELEASE-NAME-gitcall-pimp-service-account
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "pimp"
---
# Source: gitcall_pimp_release/charts/gitcall_pimp/templates/configmap.yaml
